packages/cli/src/codemods/transforms/transform-grid-options-v33-0/__fixtures__/scenarios/jsx/transformGridOptionToColDef/output.js

```diff
@@ -0,0 +1,80 @@
+function GridComponent (props) {
+  return (
+    (<AgGridReact
+      columnDefs={[]}
+      rowData={[]}
+      defaultColDef={{
+        unSortIcon: true
+      }}
+    />)
+  );
+}
+
+function GridComponent2() {
+  return (
+    (<AgGridReact
+      columnDefs={[]}
+      rowData={[]}
+      defaultColDef={{
+        unSortIcon: true
+      }}
+    />)
+  );
+}
+
+function GridComponent3() {
+  return (
+    (<AgGridReact
+      columnDefs={[]}
+      rowData={[]}
+      defaultColDef={{
+        unSortIcon: false
+      }}
+    />)
+  );
+}
+
+function GridComponent4 (props) {
+  return (
+    (<AgGridReact
+      columnDefs={[]}
+      rowData={[]}
+      defaultColDef={{
+        unSortIcon: true
+      }} />)
+  );
+}
+
+function GridComponent5() {
+  return (
+    (<AgGridReact
+      columnDefs={[]}
+      rowData={[]}
+      defaultColDef={{
+        unSortIcon: true
```

Comment:
Concerned that this is going to introduce extra renders due to defaultColDef not being memoized. If the defaultColDef is already on the grid like this then thats fine to mutate, but otherwise I feel like we need to follow our own advice  from https://grid-staging.ag-grid.com/react-data-grid/react-hooks/

Reply:
Likewise - but again the complexity of the code mod scales too much to be able to handle this, the alternative option is no code-mod for this I think, unless you have some idea?

Reply:
I think we should not attempt this code mod if we are going to leave the user with sub optimal code.